Cargo.toml

```diff
@@ -7,24 +7,24 @@ license = "Apache-2.0"
 description = "A matrix message logger with full text search support"
 
 [dependencies]
-tantivy = { git = "https://github.com/tantivy-search/tantivy", rev = "598b076" }
-tinysegmenter = "0.1"
-rusqlite = { version = "0.20", features = ["sqlcipher"] }
+tantivy = "0.11.3"
+tinysegmenter = "0.1.1"
+rusqlite = { version = "0.21.0", features = ["sqlcipher"] }
 fs_extra = "1.1.0"
-r2d2_sqlite = "0.12"
-r2d2 = "0.8"
-failure = "0.1.5"
+r2d2_sqlite = "0.13.0"
+r2d2 = "0.8.8"
+failure = "0.1.6"
 aes-ctr = "0.3.0"
 crypto-mac = "0.7.0"
 hmac = "0.7.1"
 sha2 = "0.8.1"
 hkdf = "0.8.0"
 pbkdf2 = "0.3.0"
-rand = "0.7.2"
-zeroize = "1.0.0"
+rand = "0.7.3"
```

Comment:
I'm a bit surprised that we're bumping patch releases manually? The newest patch releases will be used in new builds (or via a `cargo update`).

Reply:
I just use [cargo upgrade](https://github.com/killercup/cargo-edit) to do it automatically, we're mainly switching to a release version of Tantivy here.

Reply:
Fair enough. Its not really a problem other than it adds a bit of churn to local checkouts